elements/rh-alert/rh-alert.ts

```diff
@@ -63,9 +63,13 @@ export class RhAlert extends LitElement {
 
   @property({ reflect: true, type: Boolean }) toast = false;
 
-  @bound private _closeHandler() {
-    this.hidden = true;
-    this.dispatchEvent(new AlertCloseEvent());
+  @property({ reflect: true, type: Boolean }) dismissable = false;
+
+  #closeHandler() {
+    const event = new AlertCloseEvent();
+    if (this.dispatchEvent(event)) {
```

Comment:
We need unit tests for this, too

Reply:
should be able to just `expect(element).to.be.undefined` for that

Reply:
You would think so but I was not able to get the element to be undefined when I would run through it even though I could verify that the event was being called / not called when adding other attributes.  Figured there might have been something else I needed to do to update the component or the fixture wasn't removing it for some reason.

Reply:
you can also check [element.isConnected](https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected)

https://lit.dev/playground/#gist=b0639065ab09e87708e56a6c8d2fdc2f

Reply:
Looks like that was it, was able to remove the sinon spy